docs/virtual_machines/instancetypes.md

```diff
@@ -0,0 +1,298 @@
+# Instancetypes and preferences
+
+**FEATURE STATE:** 
+
+* `v1alpha1` (Experimental) as of the [`v0.56.0`](https://github.com/kubevirt/kubevirt/releases/tag/v0.56.0) release
+
+## Introduction
+
+KubeVirt's [`VirtualMachine`](https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachine) API contains many advanced options for tuning the performance of a VM that goes beyond what typical users need to be aware of. Users have previously been unable to simply define the storage/network they want assigned to their VM and then declare in broad terms what quality of resources and kind of performance characteristics they need for their VM.
+
+Instancetypes and preferences provide a way to define a set of resource, performance and other runtime characteristics, allowing users to reuse these definitions across multiple [`VirtualMachines`](https://kubevirt.io/api-reference/master/definitions.html#_v1_virtualmachine).
+
+## VirtualMachineInstancetype
+
+```yaml
+---
+apiVersion: instancetype.kubevirt.io/v1alpha1
+kind: VirtualMachineInstancetype
+metadata:
+  name: example-instancetype
+spec:
+  cpu:
+    guest: 1
+  memory:
+    guest: 128Mi
+```
+
+KubeVirt provides two Instancetype based [`CRDs`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/), a cluster wide [`VirtualMachineClusterInstancetype`](http://kubevirt.io/api-reference/main/definitions.html#_v1alpha1_virtualmachineclusterinstancetype) and a namespaced [`VirtualMachineInstancetype`](http://kubevirt.io/api-reference/main/definitions.html#_v1alpha1_virtualmachineinstancetype). These [`CRDs`](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/) encapsulate the following resource related characteristics of a [`VirtualMachine`](http://kubevirt.io/api-reference/main/definitions.html#_v1alpha1_virtualmachine) through a shared [`VirtualMachineInstancetypeSpec`](http://kubevirt.io/api-reference/main/definitions.html#_v1alpha1_virtualmachineinstancetypespec):
```

Comment:
Can you use `https` in all links, and can you point them to the latest API version?
For example the VirtualMachine link in this paragraph points to `v1alpha1` version, instead of `v1`:
http://kubevirt.io/api-reference/main/definitions.html#_v1alpha1_virtualmachine

Instead it should be: https://kubevirt.io/api-reference/main/definitions.html#_v1_virtualmachine

Reply:
Done